.github/workflows/android.yml

```diff
@@ -0,0 +1,29 @@
+name: Android CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
Give that you're new branch is develop and you will open all branches to develop this should change.